sdk/core/System.ClientModel/src/System.ClientModel.csproj

```diff
@@ -6,7 +6,7 @@
     <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
     <ApiCompatVersion>1.2.1</ApiCompatVersion>
     <Nullable>enable</Nullable>
-    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net6.0;net8.0</TargetFrameworks>
```

Comment:
Feels like we should be using `$(RequiredTargetFrameworks)` here and just adding on rather than defining our own special set.   

Wonder what the rationale was?

Reply:
SCM was originally created so it could be built outside our repo.

@m-redding, pls don't check this in until we make a decision regarding how many targets we should have for SCM.

Reply:
@annelo-msft would it be better for me to exempt SCM from a net8.0 target for now? Like I did for AOAI

Reply:
I would assume that SCM should follow the same target framework pattern as Azure libraries.   This is consistent with the approach advised by the .NET team, though a bit more conservative in adoption.

Reply:
The reason that we used the "6 month" heuristic on Core was to soften the potential impact of the scenario when someone was using a no-longer-supported platform where we had made specific performance tweaks which would take a performance hit in the case where we remove their explicit target and they fall back to `netstandard2.0`.    I'm not sure how much of that really applies to the first release of SCM, but there's no harm in leaving `net6.0` in until we retire it from Core in May.

For context, the .NET team had advised that we drop unsupported platforms immediately when they hit end-of-life or we follow their model of a major revision for each .NET release.  When we talked that through in the Core meeting, folks felt that was a bit more harsh than we were comfortable with because either approach would potentially scare customers from taking a new version of the library until they were able to update their runtime.

Reply:
As a v-team, we wrote down this plan for handling TFMs in Core and client libraries: [Core_library_TFM_plan.md](https://gist.github.com/annelo-msft/c62a54d455e2fd635545b3f522df53d5).

In this case, I would recommend we add `net8.0` to SCM so that our Core libraries have the same newest targets -- I think we could cause confusion and problems if SCM-based clients and Azure.Core-based clients couldn't use the same .NET features and APIs.  I would recommend removing the `net6.0` target from SCM according to the plan, i.e. in six months.  It would be good to do due-diligence ahead of time to understand the impact of removing that target to cost the work to remove it, as well as whatever communication was needed to users as a result of that.